srcs/Simulation.cpp

```diff
@@ -2,18 +2,25 @@
 
 Simulation::Simulation() :
   _population(Population(_maxPop)),
-  _clientID(simxStart("127.0.0.1", 19997, true, true, 5000, 5)),
-  _robot(Robot(_clientID)) {
+  _clientID(simxStart("127.0.0.1", 19997, true, true, 5000, 5)) {
   if (_clientID == -1) {
     throw "simxStart error";
   }
+  for (uint16_t i = 0; i < _maxRobots; ++i) {
+    _robots.push_back(Robot(_clientID));
+    usleep(100);
```

Comment:
Sometimes there is an error thrown by the robot creation function, I tought that waiting between two api calls could help but no it doesn't